### PR TITLE
edit cluster create to add all openstack ip address

### PR DESCRIPTION
**Pull request by CoderKevinZhang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/globalscheduler/test/create_clusters.sh [hunk 1]
```diff
@@ -13,46 +13,53 @@
...
+    area: $2
+    city: $3
+    country: $5
+    province: $4
+  ipaddress: $6
+  memcapacity: 256
   operator:
-    operator: "None"
-  flavors:
-    - flavorid: "1"
```
Comment (jshaofuturewei) [inline hack/globalscheduler/test/create_clusters.sh:37]: I think we need flavor capacity for scheduling.

Comment (CoderKevinZhang) [reply hack/globalscheduler/test/create_clusters.sh:37]: there is flavor capacity in yaml file
image

**Review by jshaofuturewei — APPROVED** [review]

**Comment by cathyhongzhang** [comment]
Kevin, as we discussed in the meeting, you will update the PR to incorporate my suggestions.

**Review by kimeunju108 — APPROVED** [review]
LGTM

**Comment by cathyhongzhang** [comment]
As we discussed, please modify the sample_new_1000_clusters.yaml file so that each region has 50 AZs and 1000 clusters map to 1000 region+AZ combinations. Also modify sample_1000_pods.yaml file so that each cluster has a mapping pod w.r.t. region+AZ and geolocation.

**Review by cathyhongzhang — APPROVED** [review]
LGTM

**Pull request merged**